app/views/application/_helper_functions.coffee

```diff
@@ -29,15 +29,38 @@ window.reload_checkbox = ->
       new_checkbox = new_checkbox + '<span class=\'small-2 columns end\'><label><input id=\'' + escapeHtml(param.id) + '\' style=\'margin-right:5px;\' type=\'checkbox\'>' + escapeHtml(param.label) + '</label></span> '
 
   ### append new values###
-
   $(' .moes_infos').each ->
     $(this).html new_checkbox
     return
   toastr.success("Moes refreshed")
 
+#reload and join parameters and moes which have max 10 values
```

Comment:
Opis metody nie jest poprawny - to jest raczej "Reload and join given parameters and moes", jako, że używamy ``allowed_params`` dane przez argument